### PR TITLE
add_test_subctl_benchmark

### PR DESCRIPTION
**Pull request by pinikomarov**

### CODE REVIEW

File: setup_subm.sh [hunk 1]
```diff
@@ -3581,6 +3581,18 @@ function test_project_e2e_with_go() {
 
 # ------------------------------------------
 
+function test_subctl_benchmarks() {
+  PROMPT "Testing subctl benchmark: latency and throughput tests"
+  trap_to_debug_commands;
+
+  subctl benchmark latency ${KUBECONF_CLUSTER_A} ${KUBECONF_CLUSTER_B}
+
+  subctl benchmark throughput ${KUBECONF_CLUSTER_A} ${KUBECONF_CLUSTER_B}
```
Comment (manosnoam) [inline setup_subm.sh:3590]: Do you know what are the exit codes for benchmark ?
Does it return non-zero exit code if benchmark failed ?
If so, is the any threshold that bellow it the benchmark is counted as failed ?

Comment (pinikomarov) [reply setup_subm.sh]: with my latest patch here , 
It returns non-zero exit code if benchmark is failed
about the threshold , I asked the developers , and don't believe in slecting an arbitrary threshhold limit for these tests

Comment (manosnoam) [reply setup_subm.sh]: What is the criteria for failed benchmark, and what for passed ?

**Comment by manosnoam** [comment]
Thanks Pini, I've merged your PR, and now I can see benchmark output:

```
13:11:57 subctl benchmark latency /mnt/skynet-data/pkomarov-env/pkomarov-cluster-a/auth/kubeconfig /mnt/skynet-data/pkomarov-env/ocpup/.config/cl2/auth/kubeconfig
13:12:23 MIGRATED TCP REQUEST/RESPONSE TEST from 0.0.0.0 (0.0.0.0) port 0 AF_INET to 10.139.1.40 (10.139.) port 0 AF_INET : first burst 0
13:12:23 Minimum Latency Microseconds,Mean Latency Microseconds,Maximum Latency Microseconds,Stddev Latency Microseconds,Transaction Rate Tran/s
13:12:23 18567,19067.27,43144,1544.19,52.344
13:12:23 Minimum Latency Microseconds:	18567
13:12:23 Mean Latency Microseconds:	19067.27
13:12:23 Maximum Latency Microseconds:	43144
13:12:23 Stddev Latency Microseconds:	1544.19
13:12:23 Transaction Rate Tran/s:	52.344
13:12:48 MIGRATED TCP REQUEST/RESPONSE TEST from 0.0.0.0 (0.0.0.0) port 0 AF_INET to 10.136.0.88 (10.136.) port 0 AF_INET : first burst 0
13:12:48 Minimum Latency Microseconds,Mean Latency Microseconds,Maximum Latency Microseconds,Stddev Latency Microseconds,Transaction Rate Tran/s
13:12:48 19105,19659.30,38983,1280.60,50.767
13:12:48 Minimum Latency Microseconds:	19105
13:12:48 Mean Latency Microseconds:	19659.30
13:12:48 Maximum Latency Microseconds:	38983
13:12:48 Stddev Latency Microseconds:	1280.60
13:12:48 Transaction Rate Tran/s:	50.767
13:12:48 (conda-test) 
13:12:48 subctl benchmark throughput /mnt/skynet-data/pkomarov-env/pkomarov-cluster-a/auth/kubeconfig /mnt/skynet-data/pkomarov-env/ocpup/.config/cl2/auth/kubeconfig
13:13:20 iperf3: error - unable to send control message: Bad file descriptor
13:13:20 [going to retry]
13:13:20 iperf3: error - unable to send control message: Bad file descriptor
13:13:20 [going to retry]
13:13:20 iperf3: error - unable to send control message: Bad file descriptor
13:13:20 [going to retry]
13:13:20 iperf3: error - unable to send control message: Bad file descriptor
13:13:20 [going to retry]
13:13:20 
13:13:50 iperf3: error - unable to send control message: Bad file descriptor
13:13:50 [going to retry]
13:13:50 iperf3: error - unable to send control message: Bad file descriptor
13:13:50 [going to retry]
13:13:50 iperf3: error - unable to send control message: Bad file descriptor
13:13:50 [going to retry]
13:13:50 iperf3: error - unable to send control message: Bad file descriptor
13:13:50 [going to retry]
```

**Comment by pinikomarov** [comment]
cool , 
About threshold: the dev refused to set a threshold, so this check as is if it's working that's fine by them.

I see an error here "Bad file descriptor" weird no ?thoughts ?

**Pull request merged**